frontend/src/entities/configuration/dynamic-configuration.tsx

```diff
@@ -260,482 +187,110 @@ export function DynamicConfiguration () {
         return output;
     }
 
-    /**
-     * Converts a JSON object into an outline structure represented as React nodes.
-     *
-     * @param {object} [json={}] - The JSON object to be converted.
-     * @returns {React.ReactNode[]} - An array of React nodes representing the outline structure.
-     */
-    function jsonToOutline (json = {}) {
-        const output: React.ReactNode[] = [];
-
-        for (const key in json) {
-            const value = json[key];
-            output.push((<li><p><strong>{key}</strong></p></li>));
-
-            if (isObject(value)) {
-                const recursiveJson = jsonToOutline(value); // recursively call
-                output.push((recursiveJson));
-            }
-        }
-        return <ul>{output}</ul>;
-    }
-
     const changesDiff = getJsonDifference(applicationConfig.configuration, state.form.configuration);
```

Comment:
we should probably move this and the function to the confirm-configuration-changes modal file like you did with the  jsonToOutline file. they go hand in hand and I dont think changesDiff is use anywhere else other than that modal.